src/lib/style/components/_facet-plots-card.scss

```diff
@@ -1,7 +1,8 @@
-.FacetPlotsCard {
+.FacetPlotsCard {  
   background-color: white;
+  overflow: hidden;
```

Comment:
What's the purpose of `overflow: hidden` here?

Reply:
In the worst case scenario for mobile breakpoints (like the styleguidist context, where the home page component does not take up the full width of the viewport!), this is to guarantee that the component content (like the plot legend labels, for example) do not extend outside of the card boundary.  That content will be cut off in this rare case.

Reply:
Ah, totally makes sense. Thanks!